vertx-mongo-client/src/main/java/io/vertx/ext/mongo/MongoGridFsUpload.java

```diff
@@ -0,0 +1,32 @@
+package io.vertx.ext.mongo;
+
+import io.vertx.codegen.annotations.Fluent;
+import io.vertx.codegen.annotations.ProxyGen;
+import io.vertx.codegen.annotations.VertxGen;
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Handler;
+
+@ProxyGen
+@VertxGen
+public interface MongoGridFsUpload {
```

Comment:
How would we use a WriteStream with a Service Proxy type interface? I'd prefer a Read/WriteStream because we could Pump.